src/main/java/emissary/util/search/FastBoyerMoore.java

```diff
@@ -289,13 +291,13 @@ public static void main2(final String[] args) {
                     scanner.scan(dataBytes, pos, hits);
                     for (int i = 0; i < hits.size(); i++) {
                         final int[] tmp = hits.get(i);
-                        System.out.println("Hit At:" + tmp[0] + " id: " + tmp[1]);
+                        logger.info("Hit At: {} id: {}", tmp[0], tmp[1]);
                     }
                     hits.clear();
                 }
             } catch (Exception e) {
                 System.out.println("Exception in test:" + e);
```

Comment:
Should this line be removed as well?

Reply:
Whoops, yes. Is the goal to remove all console logging?